x-pack/plugins/security_solution/public/timelines/components/timeline/search_or_filter/search_or_filter.tsx

```diff
@@ -136,6 +132,55 @@ export const SearchOrFilter = React.memo<Props>(
                 updateReduxTime={updateReduxTime}
               />
             </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup
+                direction="row"
+                gutterSize="none"
+                alignItems="center"
+                justifyContent="center"
+              >
+                <EuiToolTip
+                  content={
+                    dataProviders?.length > 0 && !isDataProviderVisible
+                      ? DATA_PROVIDER_HIDDEN_POPULATED
+                      : dataProviders?.length === 0 && !isDataProviderVisible
+                      ? DATA_PROVIDER_HIDDEN_EMPTY
+                      : DATA_PROVIDER_VISIBLE
+                  }
```

Comment:
Can we simplify this?

https://github.com/elastic/kibana/blob/3d197d36a5f9d3e936ee998a263de981a432779c/STYLEGUIDE.mdx#L401-L413

Reply:
Thanks for catching this @semd, I have simplified it. Please check it out here: https://github.com/elastic/kibana/pull/168230/files#diff-f9cee7aa12f4f7845b51077c05b406b32d0f27dbac3ce23b343e7516563cbfccR104-R114